deploy/olm/kustomization.yaml

```diff
@@ -10,4 +10,4 @@ kind: Kustomization
 images:
 - name: monitoring-stack-operator
   newName: monitoring-stack-operator
-  newTag: 0.0.2
+  newTag: 0.0.6
```

Comment:
Unrelated change?

Reply:
ya i can remove that, got update when I ran `make bundle`